main.css

```diff
@@ -106,6 +106,7 @@
   flex-grow: 0;
 }
 
+/* section paragraph */
 .headline .second-text p {
   font-family: 'Poppins', sans-serif;
   font-style: normal;
```

Comment:
- [ ] [Optional] You are using font-family: sans-serif only 'Poppins' is not used for that you need to import font from google and then you can use Poppins. In the project requirement, you need to use the Poppins font family. So kindly implement it